VariationalBayes/test_exponential_families.py

```diff
@@ -4,10 +4,16 @@
 from autograd import grad, jacobian, hessian
 import VariationalBayes.ExponentialFamilies as ef
 import VariationalBayes.Modeling as model
+
 import unittest
 import numpy.testing as np_test
 import scipy as sp
 
+from numpy.polynomial.hermite import hermgauss
+
+# global parameters for computing lognormals
+GH_LOC, GH_WEIGHTS = hermgauss(4)
```

Comment:
Sorry I keep coming back to this, but what is the reason for making these global parameters?  They're only used in one test, please just define them as local variables there.